### DISCUSSION TITLE
Version 5 plans/discussions

### DISCUSSION THREAD

**Discussion by Legorooj** [pyinstaller/pyinstaller#6155] [Features & Development]
General discussion of thoughts/plans for version 5 of PyInstaller, the next major release, which will include breaking changes.

**Comment by Legorooj** [comment]
I'd like to drop support for Python 3.6 in version 5 - it's EOL date is 23rd of December 2021. I doubt v5 will be ready by then.

**Reply by bwoodsend** [reply]
Yes I agree. Although so little changed between 3.6 and 3.7 that I doubt it'll make much difference.

**Comment by bwoodsend** [comment]
Can we make `--noconfirm` the default? Unless the bundle has been manually fiddled with, a PyInstaller build should be trivially reproducible anyway.

**Reply by Legorooj** [reply]
Yeah sure.

**Comment by Legorooj** [comment]
#6460
